### DISCUSSION TITLE
How to autoexpand on certain filetypes?

### DISCUSSION THREAD

**Discussion by alexmozaidze** [dcampos/nvim-snippy#110] [Q&A] [answered]
I want so that when opening a non-existant file (like `.editorconfig`) it autofills the file with a template (inspired by Doom Emacs [file-templates](https://docs.doomemacs.org/latest/modules/editor/file-templates/) module). Is there any way to achieve that?

**Comment by dcampos** [comment, marked as answer]
Currently Snippy doesn't provide that functionality, but I've added a [wiki page](../../nvim-snippy/wiki/File-templates) that gives an example of how I do it. I hope it will be helpful.

**Reply by alexmozaidze** [reply]
Works wonders, thank you! It's a shame it's not baked into the plugin as a feature, it's very useful
